client/src/components/SubmissionTabs/SubmissionTabs.tsx

```diff
@@ -68,15 +62,15 @@ function SubmissionTabs({ card }: cardProps): JSX.Element {
         <Box p={3}>
           <SubmissionTabPanel value={value} index={0}>
             <Grid container spacing={5}>
-              {card.map((data, key) =>
+              {card.map((data) =>
                 data.imageFiles.length > 1 ? (
                   data.imageFiles.map((image) => (
-                    <Grid item md={3} xs={12} key={key}>
+                    <Grid item md={3} xs={12} key={data.contestID + image}>
```

Comment:
Used the combination of the contestId and the images in each imageArray as a key